src/server/img_storage.rs

```diff
@@ -54,35 +121,72 @@ pub struct StoreResult {
     pub url: uri::Uri,
     pub hash: String,
```

Comment:
Might wanna also kill these unused fields (like `hash`) until they're needed.

Another general nit -- I've been trying not to include "Result" in struct names such as this one as it makes them sound like they're a variant of a `Result` (such as `HandlerResult`) to fresh eyes. Maybe `StoredImage`